### ISSUE TITLE
got CLUSTER ALREADY SPAWNED  when  respawnAll or respawn

### ISSUE THREAD

**Issue by zeteticl** [meister03/discord-hybrid-sharding#77]
**Describe the bug**
got CLUSTER ALREADY SPAWNED  when  respawnAll or respawn
```Uncaught Error Error: CLUSTER ALREADY SPAWNED | ClusterId: 0
discord-hybrid-sharding\dist\Core\Cluster.js:113:19)
discord-hybrid-sharding\dist\Core\Cluster.js:184:21)
```

(**To Reproduce**)

**Expected behavior**
using client.cluster.respawnAll({ clusterDelay: 5000, respawnDelay: 5500, timeout: 30000 })

**Screenshots & Error Stack**

**Discord.js Version**

-   [Give your Discord.js Version] : discord.js@^14.7.1

**Additional context**
Maybe HeartbeatManager have same problem, so sometime the server crash.

**Comment by meister03** [comment]
no, its because there is an if(this.thread) condition and it does not gets reset if you kill the cluster...

**Comment by zeteticl** [comment]
So how can i use respawn or respawnAll function...

**Comment by meister03** [comment]
Fixed with version 2.1.1
